R/tmb-model.R

```diff
@@ -11,26 +11,6 @@ prepare_tmb_inputs <- function(naomi_data) {
   stopifnot(is(naomi_data, "naomi_data"))
   stopifnot(is(naomi_data, "naomi_mf"))
 
```

Comment:
@jeffeaton I have removed this as it is overwritten by the `create_anc_Amat()` function that follows.